shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/utils/networking/OpenshiftNetworkingTestUtils.java

```diff
@@ -61,4 +68,12 @@ public void patchNetworkResource(String name, String namespace) {
     public void cleanUp() {
         client.routes().inAnyNamespace().delete();
     }
+
+    private void patchOpenshiftIngressDomain() {
+        Ingress openshiftIngress = new IngressBuilder()
+                .withMetadata(new ObjectMetaBuilder().withName(OpenshiftNetworkingService.CLUSTER_DOMAIN_RESOURCE_NAME).build())
+                .withSpec(new IngressSpecBuilder().withDomain("apps.openbridge-dev.fdvn.p1.openshiftapps.com").build())
```

Comment:
@r00ta hardcoded domain? is this correct?

Reply:
It's in the tests @kostola , whatever value is ok

Reply:
ok so it's not actually being called

Reply:
if this is not being called..the PR is fine for me :+1:

Reply:
Well it is called indirectly. The openshift mock comes with no resources, and this is the resource where OCP stores the domain address. 
With this PR we do retrieve this domain from this resource and we use it to specify the hostname for the ingress. If we don't create it, our controller would not be able to fetch this resource and would fail to create the route